frontend/vite.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react-swc'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
```

Comment:
Lets add svg plugin and define project root (with @ icon for imports from src folder) for frontend to ease development in future - 

```
import svgx from '@svgx/vite-plugin-react';
import { resolve } from 'path';
    
plugins: [react(), svgx()],
    resolve: {
      alias: [
        { find: '@', replacement: resolve(__dirname, './src') } ]
      }, ...
```